core/src/testFixtures/java/bio/terra/pearl/core/factory/participant/EnrolleeFactory.java

```diff
@@ -46,10 +46,11 @@ public Enrollee buildPersisted(String testName) {
         return enrolleeService.create(builderWithDependencies(testName).build());
     }
 
-    public Enrollee buildPersisted(String testName, UUID studyEnvironmentId, UUID participantUserId) {
+    public Enrollee buildPersisted(String testName, UUID studyEnvironmentId, UUID participantUserId, UUID profileId) {
```

Comment:
maybe instead of adding a paramter to this function, create a new one, and have the existing one delegate?  So we don't have to specify "null" in a bunch of places where we don't care about the profile?

Reply:
nvm, it looks like this wasn't used elsewhere